packages/webview_flutter/webview_flutter_platform_interface/lib/src/platform_webview_controller.dart

```diff
@@ -259,6 +259,14 @@ abstract class PlatformWebViewController extends PlatformInterface {
     throw UnimplementedError(
         'setUserAgent is not implemented on the current platform');
   }
+
+  /// Toggle the listener for content offset changes. Set to null to stop listening.
+  Future<void> setOnContentOffsetChanged(
```

Comment:
When looking forward to how the iOS implementation will need to be done, I noticed the values for `oldLeft` and `oldTop` won't be available. The iOS implementation will only be able to listen for `left` and `top` by observing [UIScrollView.contentOffset](https://developer.apple.com/documentation/uikit/uiscrollview/1619404-contentoffset?language=objc). So the interface method should probably not include  `int oldLeft` or `int oldTop`. It should also probably pass a class that can be extended by a platform: (e.g. 

```dart
class ContentOffsetChange {
  ContentOffsetChange(this.x, this.y);

  final x;
  final y;
}
```